source/sgp_mode/Instructions.h

```diff
@@ -96,6 +96,25 @@ INST(IO, {
   *a = next;
   state.input_buf.push(next);
 });
+
+INST(Reuptake, {
+  uint32_t next;
+  float score = state.world->GetTaskSet().CheckTasks(state, *a);
```

Comment:
Let's put this section of repeated code (repeated from IO) into a small helper method that you can then call from both instructions